renpy/loader.py

```diff
@@ -243,6 +248,40 @@ def index_archives():
                     except:
                         raise
 
+    global paths_renpy_common
```

Comment:
I don't like this function. There are too many constants and byte strings - I don't know what  b"\x50\x4b\x03\x04" means here. 

I'd suggest the following changes:

* Let's not do a search for renpy.zip in sys.path. I think it's better to simply check for it in config.renpy_base, and if it doesn't exist, that's it. 
* https://github.com/renpy/renpy-build/blob/master/runtime/android/apk.py#L134 does something very similar, without knowing as much about how a zip file works. I think this could be based off that.